src/sql/src/plan/statement.rs

```diff
@@ -404,6 +406,9 @@ pub struct StatementContext<'a> {
     /// The types of the parameters in the query. This is filled in as planning
     /// occurs.
     pub param_types: RefCell<BTreeMap<usize, ScalarType>>,
+    /// Whether the statement contains an expression that can make the exact column list
+    /// ambiguous. For example `NATURAL JOIN` or `SELECT *`.
```

Comment:
```suggestion
    /// ambiguous. For example `NATURAL JOIN` or `SELECT *`. This is filled in as planning occurs.
```

Very useful to have documented that this doesn't get set to its final value by the scx constructor.

Reply:
Added